src/App.js

```diff
@@ -1,19 +1,48 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
+import ChatLog from './components/ChatLog';
 import chatMessages from './data/messages.json';
 
 const App = () => {
+
+  const [entries, setEntries] = useState(chatMessages);
+
+  const toggleLike = (id) => {
```

Comment:
I know that we did together in my office hours, but please reach out if you need any further explanations as to **how and why** we want to do this. Great job on following the implementation! 

The (Usual) Steps to Lifting Up State: 

1. Establish state with data (from data folder or API) in necessary parent component.
2. Design `updateFunction` that uses the `id` of an object to preforms desired changes and updates state.
3. Pass `updateFunction` to necessary children components
4. Make `updateFunction` the event handler and passing `prop.id` as parameter.